tests/unit/dataactbroker/test_dashboard_handler.py

```diff
@@ -208,6 +213,9 @@ def setup_submissions(sess, admin=False):
     sub1_ab2 = ErrorMetadata(job=sub1_ab, original_rule_label=rule_ab2.rule_label, occurrences=130,
                              file_type_id=rule_ab2.file_id, target_file_type_id=rule_ab2.target_file_id,
                              rule_failed=rule_ab2.rule_error_message, severity_id=rule_ab2.rule_severity_id)
+    sub1_cd1 = ErrorMetadata(job=sub1_cd1, original_rule_label=rule_cd1.rule_label, occurrences=10,
```

Comment:
This has the same name as the new job (which shouldn't be an issue once the comment on the job is handled but is probably still bad practice)